src/templates/sermons.js

```diff
@@ -3,12 +3,30 @@ import Layout from "../components/layout"
 import './sermons.css'
 
 export default function Sermons({ pageContext }) {
-    const sermons = pageContext.nodes;
+    const sermons = pageContext.nodes,
+        {total, current} = pageContext.pagination,
+        pageStart = Math.max(1, current - 10),
```

Comment:
We can nix the magic number here if we give it a name, something like `showAdjacentPages = 10`. That helps indicate what the `10`s on lines 8 + 9 are doing.